tests/migrations/test_runner.py

```diff
@@ -56,3 +57,29 @@ def test_version() -> None:
     assert runner._get_next_version(migration_key) == 2
     runner._update_migration_status(migration_key, Status.COMPLETED)
     assert runner._get_next_version(migration_key) == 3
+
+
+def test_no_schema_differences() -> None:
+    from snuba.datasets.schemas.tables import TableSchema
+    from snuba.datasets.storages import StorageKey
+    from snuba.migrations.parse_schema import get_local_schema
```

Comment:
Why are these imported in the method ? Not sure I get if there is a circular dependency.

Reply:
There isn't a circular dependency, I forgot to move it back up